discord/message.py

```diff
@@ -1601,9 +1601,14 @@ async def edit(
         if view is not MISSING:
             self._state.prevent_view_updates_for(self.id)
             payload["components"] = view.to_components() if view else []
+            if view and view.is_v2():
```

Comment:
Rename `is_v2` to `is_components_v2`

Reply:
ur late

Reply:
yk how long it fucking takes to do a full pr review while being half asleep LMAO